test.array.js

```diff
@@ -0,0 +1,25 @@
+var assert = require('assert');
```

Comment:
This include assumes that a node-style module system is available and will require some extra tooling to work in the browser, however we can work around it here for now. `chai` (included below) is an assertion library that is added to the global scope by the runner below...